src/libraries/Microsoft.PowerFx.Core/Functions/UserDefinedFunction.cs

```diff
@@ -148,6 +152,27 @@ public override IEnumerable<StringGetter[]> GetSignatures()
             return new[] { _args.Select<UDFArg, TexlStrings.StringGetter>(key => _ => key.NameIdent.Name.Value).ToArray() };
         }
 
+        public (IntermediateNode topNode, ScopeSymbol ruleScopeSymbol) GetIRTranslator()
+        {
+            return IRTranslator.Translate(_binding);
```

Comment:
>_binding [](http://example.com/codeflow?start=42&length=8)

Can we add null check on binding and add msg, First Call BindBody()

Reply:
`IRTranslator.Translate` already calls `Contracts.AssertValue(binding);`.